torch/_inductor/mkldnn_lowerings.py

```diff
@@ -431,7 +431,7 @@ def qconvolution_binary(
             unary_algorithmm,
         ):
             if (
-                binary_attr == "sum"
+                (binary_attr == "sum" or binary_attr == "add")
```

Comment:
Feels binary add post op of oneDNN support different input data type of `add` (perhaps better perf to do it inside onednn)

Reply:
Modified, thanks for your comments!